package.json

```diff
@@ -23,11 +25,16 @@
     "express-validator": "*",
     "mongodb": "*",
     "mongoose": "*",
+    "mongoose-unique-validator": "*",
+    "nodemailer": "*",
     "passport": "*",
     "passport-http": "*",
     "passport-local": "*",
     "qrcode-js": "0.0.2",
-    "sharp": "^0.17.3",
-    "nodemailer": "*"
+    "sharp": "^0.17.3"
+  },
+  "devDependencies": {
+    "vows": "*",
```

Comment:
It is not always a good idea to have the latest version for dependencies. You usually would want to verify that there are no breaking changes in the dependencies before upgrading.